src/components/ReportActionsSkeletonView/SkeletonViewLines.tsx

```diff
@@ -1,26 +1,20 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {Circle, Rect} from 'react-native-svg';
 import SkeletonViewContentLoader from '@components/SkeletonViewContentLoader';
 import styles from '@styles/styles';
 import themeColors from '@styles/themes/default';
 import CONST from '@src/CONST';
 
-const propTypes = {
-    /** Number of rows to show in Skeleton UI block */
-    numberOfRows: PropTypes.number.isRequired,
-    shouldAnimate: PropTypes.bool,
+type SkeletonViewLinesProps = {
+    numberOfRows: 1 | 2 | 3;
```

Comment:
you sure that those are only values which we can pass? 
there are conditions to check if this props is > than 1 or 2 so I think we can pass something bigger than 3?

Reply:
If you look at `ReportActionsSkeletonView/index.tsx` you can see there are only 3 values passed (1, 2, 3). You can also take a look at `HEIGHT_FOR_ROW_COUNT`, that only accepts values 1 through 3, it just made sense to me to type it that way, let me know what you think 😄